### DISCUSSION TITLE
PNG - how do I get this for any given map that is not already created and listed here?

### DISCUSSION THREAD

**Discussion by Torqueitup** [KillBait/FS19_AutoDrive_MapImages#20] [General]
How is the PNG map generated from someones FS22 map?
Can the steps to create it be listed here please?

**Comment by KillBait** [comment]
It's not complicated, but it involves a few steps.

1) download/install the map.
2) create a new game in FS22 using the new map
3) save the map from FS22 in-game menu to create a new AutoDrive config for that map

The next step might require a bit of searching, not all map makers use the same folder names/structures

4) open the map .zip file and find overview.dds and map.dem, copy them somewhere to a local disk

Everything else from now on is done in the editor

5) run the editor and load the new save game config, ignore the map image/heightmap warnings if the editor complains.
6) select the menu "Map Image->Import FS22 PDA Image" and select the overview.dds on your local disk from step 4.
7) After a pause the image should show up.
8) If the map image appears, select the menu item "Map Image->Save Imported Image" and save it to the location the editor selects
9) check in the bottom right corner to see if the heightmap was loaded ( it should say "HeightMap: Loaded" in green text
10) if the heightmap is not loaded do step 10a, if it is loaded do step 10b.

       10a) select the menu item "HeightMap->Load HeightMap" and select the map.dem you got from step 4. If no errors
               pop up, select the menu item "Heightmap->Save As Default Heightmap" and save it to the location the editor selects

       10b) select the menu item "Heightmap->Save As Default HeightMap" and save it to the location the editor selects

That's it rearly, it looks like a lot of steps, but once you do it a few times it only takes about a few minutes to do.

[NOTE] you must have a valid config loaded for the new map, the editor uses it to figure out the maps name and what to call
the folder/directory for the images to save them, if you use a different config you will overwrite that maps files with your new ones.

**Reply by Torqueitup** [reply]
Excellent and as you say, only takes a few minutes.
Thanks very much for the walk through.

**Reply by bclompen** [reply]
How can i see where there are fields on the map so i can make a course for them?

**Comment by raznzsnasna** [comment]
Hello.
how IT works for FS19?
The map Ebelsbach
https://www.modhoster.de/mods/ebelsbach-project17--2?page=4#changelog
have Not the files.
Thank you
